test/externalTests/plugins/testCommon.js

```diff
@@ -215,60 +204,73 @@ function inject (bot) {
     console.log(message)
   }
 
-  function fly (delta, cb) {
-    bot.creative.flyTo(bot.entity.position.plus(delta), cb)
+  async function fly (delta) {
+    return bot.creative.flyTo(bot.entity.position.plus(delta))
   }
 
-  function tellAndListen (to, what, listen, done) {
-    function chatHandler (username, message) {
-      if (username === to && listen(message)) {
-        bot.removeListener('chat', chatHandler)
-        done()
+  async function tellAndListen (to, what, listen) {
+    return new Promise(resolve => {
```

Comment:
doable with `once`

Reply:
The event is not listening for one message, it is listening for the first which fits the pattern. Once is not applicable.